lib/transpose_cases.rb

```diff
@@ -0,0 +1,45 @@
+class TransposeCase < OpenStruct
+  def test_name
+    "test_#{description.tr(' ', '_')}"
+  end
+
+  def skipped
+    index.zero? ? '# skip' : 'skip'
+  end
+
+  def workload
+    'Transpose.transpose(input)'
+  end
+
+  def expect
+    [
+      "<<-EXPECTED.gsub(/^ {6}/, '')",
+      indent_lines(expected),
+      indent_line('EXPECTED')
```

Comment:
There could be. :)  You can make one.  That is what I was saying. The name would probably be `left_trim` though to be clear.